express/code/blocks/pricing-cards-v2/pricing-cards-v2.js

```diff
@@ -79,19 +79,20 @@ async function getPriceElementSuffix(placeholderArr, response) {
 // eslint-disable-next-line default-param-last
 function handlePriceToken(pricingArea, priceToken = YEAR_2_PRICING_TOKEN, newPrice, priceSuffix = '') {
   try {
-    const elements = pricingArea.querySelectorAll('p');
-    const year2PricingToken = Array.from(elements).find(
-      (p) => p.textContent.includes(priceToken),
-    );
-    if (!year2PricingToken) return;
-    if (newPrice) {
-      year2PricingToken.innerHTML = year2PricingToken.innerHTML.replace(
-        priceToken,
-        `${newPrice} ${priceSuffix}`,
-      );
-    } else {
-      year2PricingToken.textContent = '';
-    }
+    const paragraphs = Array.from(pricingArea.querySelectorAll('p'))
+      .filter((p) => p.textContent.includes(priceToken));
+    if (paragraphs.length === 0) return;
+    const hasPrice = newPrice !== undefined && newPrice !== null && `${newPrice}` !== '';
```

Comment:
this line might be a bit verbose? If we are checking falsey value but not 0, we can potentially do something like `newPrice === 0 || !!newPrice`?